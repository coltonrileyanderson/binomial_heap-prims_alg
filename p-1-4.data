6 11 59 ; 9 9 194 ; 5 8 447 ; 2 4 105 ; 2 5 64 ; 
15 8 205 ; 0 14 8 ; 5 15 147 ; 11 13 489 ; 11 8 148 ; 
6 7 100 ; 15 1 351 ; 8 1 148 ; 12 10 110 ; 1 2 432 ; 
10 3 391 ; 8 3 198 ; 8 15 123 ; 5 8 396 ; 14 1 443 ; 
5 13 302 ; 7 0 120 ; 8 13 151 ; 2 15 315 ; 4 3 72 ; 
7 10 409 ; 11 1 3 ; 10 8 380 ; 2 4 236 ; 3 11 97 ; 
1 13 15 ; 1 5 49 ; 14 12 392 ; 13 12 344 ; 1 8 362 ; 
11 5 158 ; 12 13 289 ; 4 13 412 ; 8 5 230 ; 3 2 29 ; 
0 14 160 ; 5 15 77 ; 
