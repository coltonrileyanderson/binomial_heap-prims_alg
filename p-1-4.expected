0: 6
1: 7(6)100 11(6)59
2: 0(7)120 1(11)3 8(11)148
3: 5(1)49 13(1)15 14(0)8
4: 2(5)64 12(13)344 15(5)147
5: 3(2)29 10(12)110
6: 4(3)72
weight: 1268
